glossary/README.md

```diff
@@ -0,0 +1,7 @@
+# Glossary
+This folder contains terms used in the Compose project to describe certain concepts.
+
+[Application](Application.md)\
```

Comment:
I say we drop the links because we will forget to update the list for sure. The link to Manifest is missing already.

Reply:
Can this be auto-generated somehow?

Reply:
I'm dropping the links for now, and I'll look into a solution for generating them. I could only find a solution for generating a table of contents within a single file when I did a quick search